go.mod

```diff
@@ -3,8 +3,16 @@ module github.com/betterstackhq/terraform-provider-logtail
 go 1.16
 
 require (
+	github.com/agext/levenshtein v1.2.3 // indirect
+	github.com/golangci/golangci-lint v1.53.3
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
-	github.com/hashicorp/terraform-plugin-docs v0.4.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.1
-	golang.org/x/net v0.0.0-20210614182718-04defd469f4e
+	github.com/hashicorp/terraform-plugin-docs v0.15.0
+	github.com/hashicorp/terraform-plugin-go v0.17.0 // indirect
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.27.0
```

Comment:
Honestly, I would sleep better if we were a bit more conservative with the updates here, but is the tests pass, it's probably alright